lib/splitio-api-binding.js

```diff
@@ -53,11 +54,40 @@ class SplitioApiBinding {
     }
   }
 
+  /**
+   * Polls the Split.io API until since and till timestamps are the same.
+   */
+  async getAllChanges ({ path }) {
+    let since = SINCE_VALUE_FOR_FIRST_REQUEST
+    let results = await this.httpGet({ path, since })
+    const allChanges = [results]
+    // The till value represents the timestamp of the last change included in the response.
+    let till = results.till
+    while (since !== till) {
```

Comment:
does this work?

```js
const since = SINCE_VALUE_FOR_FIRST_REQUEST
const allChanges = []
while (true) {
  const results = await this.httpGet({ path, since })
  if (since === results.till) {
    break
  }
  allChanges.push(results)
  since = results.till
}
```

Reply:
ah yes, so much nice